src/main/java/com/google/maps/DirectionsApiRequest.java

```diff
@@ -133,7 +138,8 @@ public DirectionsApiRequest arrivalTime(ReadableInstant time) {
   }
 
   /**
-   * Set the departure time for a Transit directions request. If not provided, "now" is assumed.
+   * Set the departure time for a transit or driving directions request. If not provided, "now" is
```

Comment:
This seems incongruous with the comment on 57? That you need to specify departure_time?